backend/src/resources/todos/todo.router.ts

```diff
@@ -1,13 +1,28 @@
 import { NextFunction, Request, Response, Router } from 'express';
 import { Todo } from './todo.model';
 
-const todos: Todo[] = [];
+let todos: Todo[] = [];
 const todosRouter = Router();
 
 todosRouter.get('/', (req: Request, res: Response, next: NextFunction) => {
-  res.json({
-    message: 'Todo: Implement',
-  });
+  res.json(todos);
+});
+
+todosRouter.delete('/del', (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Remove `del` from the endpoint, and use an `id` route parameter instead. Access it using `req.params.id`